docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -0,0 +1,737 @@
+
+Coast
+=============
+Commanding a coast means that the module allows its rotor to slowly decelerate and come to a stop. It is a safe way to stop the module when testing since it does not cause the module to rapidly decelerate.
+
+PWM
+========
+This tells the module to spin with a percentage of its input voltage. For example, if your input voltage is 20V, and you set this parameter to 0.5, the module will spin with a 10V drive voltage, since 50% of 20V is 10V. A positive PWM percentage always goes in the counterclockwise direction, and a negative PWM percentage always goes clockwise.
+
+Velocity
+=============
+This commands the module to try and maintain a specific velocity in rad/s, using closed loop PID control. A positive velocity always goes in the counterclockwise 
+direction, and a negative velocity always goes clockwise. To learn more about velocity and voltage control 
+see :ref:`manual_velocity_control_mechanisms`
+
+Voltage
+=============
+This commands the module to a specific drive voltage. A positive voltage always goes in the counterclockwise direction, and a negative voltage always goes clockwise.
+
+ESC Input
+=============
+This commands the module to spin in the same way that a typical hobby protocol used by a flight controller would, i.e. it sends an IQUART throttle command. 
+So, setting this to 0.5 will send the module a 50% throttle command with the configuration specified in this example. However, because this 
+test is meant to simulate commands from a flight controller, it is affected by the Advanced Arming feature. That means that setting this 
+parameter will not cause the module to spin until the module has armed. **By default Vertiq speed modules require 10 consecutive throttle 
+commands between 0% and 7.5% to arm. So, to arm your module using the ESC Input parameter, set the ESC Input to 0.05 to send a 5% throttle 
+command, and click the set arrow 10 times**. On the 10th time you click the set arrow, the module should play its 2 note arming song, and begin spinning. 
+Now that the module is armed, any new throttle commands sent using the ESC Input parameter will change how it is spinning. For example, setting ESC 
+Input to 0.5 should set the module to the same speed as setting the Voltage parameter to 2.5V. This is because the module is in Voltage mode with a 
+Max Volts of 5V, so a 50% throttle command will set the module to spin with a drive voltage of 2.5V. For more information on how to configure 
+the module to properly interpret throttle commands, see the Throttle Commands and Modes section of the Feature Reference Manual. For more information on 
+arming and disarming the module, refer to the Advanced Arming section of the Feature Reference Manual.
+
+Try it Yourself
+==================
+Try out some of these parameters and observe how the module spins. For example, try setting the Voltage to 2.5V, then set the ESC Input to 0.5. 
```

Comment:
This is copied over from the old page, but it doesn't make sense anymore. On that page, I had stepped through a specific setup example with a certain direction, 2D/3D FC mode, and max volts, which made these make sense. There's no example setup prior to this here though, so these commands wouldn't actually do what they say we would unless someone happened to set it up perfectly. It also assumes a speed module.

Having an example here is good, but you may want to re-focus it on common things like Voltage and Velocity instead of ESC Input for this page. This kind of example requires more specificity

The previous page you took this from was specifically focused on walking you through a speed module setup, which isn't really the case anymore with this page. This specific example may make more sense on a getting started with speed page. Or, if you really want to keep it as it is you'd need to at least specify what your direction and max volts configurations are, and specify you are using a speed module